### DISCUSSION TITLE
How to disable lsp snippets in html and enable coc instead?

### DISCUSSION THREAD

**Discussion by tonmoy998** [NvChad/NvChad#2470] [General]
I am currently working in bootstrap so I need bootstrap class in html file.Coc html-css provides the feature

**Comment by tonmoy998** [comment]
How to solve this?
lsp vs coc
![issue](https://github.com/NvChad/NvChad/assets/74363531/ac359d0e-5b6f-4da8-b041-3a74c6f3b9f6)

**Reply by siduck** [reply]
@tonmoy998 integrating coc is a huge task, you will have to remove many plugins. 

use a plugin like nvim-html-css

```lua
{
    "hrsh7th/nvim-cmp",
    opts = {
      sources = {
        -- trigger_characters is for unocss lsp
        { name = "nvim_lsp" },
        { name = "path" },
        { name = "luasnip" },
        { name = "buffer" },
        {
          name = "html-css",
          option = {
            enable_on = {
              "html",
            }, -- set the file types you want the plugin to work on
            file_extensions = { "css", "sass", "less" }, -- set the local filetypes from which you want to derive classes
            style_sheets = {
              -- example of remote styles, only css no js for now
              "https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css",
              "https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css",
            },
          },
        },
        { name = "nvim_lua" },
      },
    },
  },

  {
    "Jezda1337/nvim-html-css",
    config = function()
      require("html-css"):setup()
    end,
  },
  ```
  
  https://github.com/Jezda1337/nvim-html-css

**Comment by tonmoy998** [comment]
even though I dont have any error.The plugin not works for me.
![2023-10-31T17:22:19,650103184+06:00](https://github.com/NvChad/NvChad/assets/74363531/1a23f35f-f90c-44bc-bc79-236342181d50)
![2023-10-31T17:22:13,706251253+06:00](https://github.com/NvChad/NvChad/assets/74363531/e17f0974-4ada-40b6-8b18-6c0c0caa1636)

**Comment by tonmoy998** [comment]
I solved this issue.It is the problem of the plugin it requires a json file inside the root dir
